pkg/deviceshifu/deviceshifuhttp/deviceshifuhttp_test.go

```diff
@@ -186,3 +195,236 @@ func TestCreatehttpURIStringNoQuery(t *testing.T) {
 		t.Errorf("createdQuery '%v' is different from the expectedQuery '%v'", createdURIString, expectedURIString)
 	}
 }
+
+// This function is for one method debug using to initialize new device and get some local var have values
+// func initTestEnv() {
+// 	err := GenerateConfigMapFromSnippet(MockDeviceCmStr, MockDeviceConfigFolder)
+// 	if err != nil {
+// 		klog.Errorf("error when generateConfigmapFromSnippet, err: %v", err)
+// 		os.Exit(-1)
+// 	}
+// 	deviceShifuMetadata := &deviceshifubase.DeviceShifuMetaData{
+// 		Name:           "TestStart",
+// 		ConfigFilePath: "etc/edgedevice/config",
+// 		KubeConfigPath: deviceshifubase.DeviceKubeconfigDoNotLoadStr,
+// 		Namespace:      "TestStartNamespace",
+// 	}
+
+// 	New(deviceShifuMetadata)
+// }
```

Comment:
remove?

Reply:
this is useful when want to debug one method for now, I prefer remain it for a while, until I solve the init problem in test case. I'll add a TODO